### ISSUE TITLE
Add CLI option to ignore all freshness checks (for selected steps)

### ISSUE THREAD

**Issue by JSKenyon** [caracal-pipeline/stimela2#341]
Currently, users need to manually delete outputs for the steps they want to rerun if those steps involve freshness checks. This can be quite cumbersome when developing/testing. A possible solution would be adding a `-f/--force` option. This would also be useful in cases where a run was interrupted by ctrl-c, leaving partially initialized outputs behind.

**Comment by landmanbester** [comment]
Wholeheartedly seconded

**Comment by o-smirnov** [comment]
Fixed on the ``issue-152`` branch.

@JSKenyon, you wanted do do it on a per-step basis -- I wonder if the ``-e/--enable-step`` option should not force ``skip_if_outputs`` to None always?

**Comment by JSKenyon** [comment]
That sounds sensible to me but it may not be necessary if a combination of `-s` and `-f` allows a number of steps (including a single step) to be forcefully rerun.

**Comment by o-smirnov** [comment]
True. The only use case this would enable is "force-refresh this particular step, but feel free to reuse outputs for other ones". Maybe we shouldn't worry about it, or deal with it holistically in #369.
